metrics/all_kind.go

```diff
@@ -93,6 +93,11 @@ func (a *All) IncErrorsStreaming(routeId string) {
 	a.codaHale.IncErrorsStreaming(routeId)
 
 }
+func (a *All) ObserveRouteKubeResources(ingress string, routeId string, service string) {
```

Comment:
this could be shortened into:
```suggestion
func (a *All) ObserveRouteKubeResources(ingress, routeId, service string) {
```